### ISSUE TITLE
gestione "trasferisci competenza"

### ISSUE THREAD

**Issue by fairizsto** [UniversitaDellaCalabria/uniTicket#108]
Quando un ufficio ha abbandonato la competenza di un ticket risulta impossibile riassegnarla in un secondo momento, nonostante il sistema confermi il trasferimento di competenza.
Nello specifico, stiamo provando ad usare questa funzione per assegnare il ticket ad una azienda esterna per la collaborazione; quando questa ha eseguito la lavorazione dovrebbe abbandonare la competenza.
Se l'ufficio "committente" ritiene che la lavorazione non sia stata correttamente eseguita ho provato a riassegnare la competenza all'azienda esterna; il trasferimento sembra avvenire con successo ma quell'ufficio non vede ugualmente il ticket.
Tale comportamento era previsto?

**Comment by francesco-filicetti** [comment]
Ciao @fairizsto ,
uniTicket non permette di riassegnare un ticket a un ufficio che ne ha precedentemente abbandonato la competenza.
Il tentativo di riassegnazione dovrebbe generare un alert con un messaggio di errore di questo tipo "La richiesta è già di competenza dell'ufficio XXX".
Confermi?

**Comment by fairizsto** [comment]
Esatto, e quell'ufficio (che ha abbandonato la competenza) non lo vede più. Nel nostro test volevamo provare a riassegnare il ticket a quell'ufficio (ditta esterna che aveva abbandonato la competenza) quando questo non aveva ben eseguito il lavoro

**Comment by francesco-filicetti** [comment]
In effetti il messaggio non è molto pertinente, meglio se fosse "Impossibile riassegnare la competenza all'ufficio XXX".
uniTicket non permette la riassegnazione di una richiesta a un ufficio che ha abbandonato la competenza (questo per evitare un effetto ping-pong)
